ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1604,6 +1604,19 @@ lldp:
   run: < true | false >
 ```
 
+### MCS client
```

Comment:
please fix this section to show the syntax. Not an example. More or less what you have in the PR description.